@navikt/core/react/src/date/monthpicker/MonthPickerInput.tsx

```diff
@@ -0,0 +1,135 @@
+import cl from "clsx";
+import { Calender } from "@navikt/ds-icons";
+import React, { forwardRef, InputHTMLAttributes, useContext } from "react";
+import { Button } from "../../button";
+import { MonthPickerContext } from "./MonthPicker";
+import { BodyShort, ErrorMessage, Label } from "../..";
+import { FormFieldProps, useFormField } from "../../form/useFormField";
+import { omit } from "../../util";
```

Comment:
Bruker selv `organise imports`-featuren i vscode ofte for å sikre relativt lik import-rekkefølge